pkg/controller/clusterdeployment/clusterdeployment_controller.go

```diff
@@ -48,10 +48,11 @@ import (
 var log = logf.Log.WithName("controller_clusterdeployment")
 
 const (
-	controllerName                = "clusterdeployment"
-	ClusterDeploymentManagedLabel = "api.openshift.com/managed"
-	hiveRelocationAnnotation      = "hive.openshift.io/relocate"
-	hiveRelocationOutgoingValue   = "outgoing"
+	controllerName                  = "clusterdeployment"
+	ClusterDeploymentManagedLabel   = "api.openshift.com/managed"
+	hiveRelocationAnnotation        = "hive.openshift.io/relocate"
+	hiveRelocationOutgoingValue     = "outgoing"
+	fakeClusterDeploymentAnnotation = "hive.openshift.io/fake-cluster"
```

Comment:
can we maybe import that constant from the hive package?
Just to not repeat ourself
https://github.com/openshift/hive/blob/697460baf215c1231db81329d57311d137fad456/pkg/constants/constants.go#L321

Reply:
But I see we are doing also for other hive labels...

Reply:
that's a good idea. i wasn't aware of that package when i put `hiveRelocationAnnotation` and `hiveRelocationOutgoingValue` in here originally